internal/pkg/daemon/selinuxprofile/selinuxpolicy_controller.go

```diff
@@ -96,6 +103,21 @@ type ReconcileSP struct {
 	record         event.Recorder
 }
 
+// Name returns the name of the controller.
+func (r *ReconcileSP) Name() string {
+	return "seccomp-spod"
```

Comment:
Nit: I guess this should say selinux-spod?

Reply:
Yep, fixed as suggested :+1: